src/App.css

```
@@ -237,30 +238,98 @@ body {
 }
 
+.unified-header-title {
+  display: flex;
+  align-items: baseline;
+  gap: 12px;
+}
+
+.unified-header-title h1 {
+  margin: 0;
+  font-size: 1.25rem;
+  font-family: "Syne", "Space Grotesk", sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Syne" (font-family-name-quotes)

(font-family-name-quotes)